lib/postgres/vacuum/monitor/version.rb

```diff
@@ -3,7 +3,7 @@
 module Postgres
   module Vacuum
     module Monitor
-      VERSION = '0.13.1'
+      VERSION = '0.14.0'
```

Comment:
I'm a little torn on this, technically the deprecation is a breaking change. But v0 is allowed to make breaking changes. I'm fine just bumping the version like this 😄

Reply:
v0 _forever_, looks like prior deprecations have been released like this as well so I'll keep it for consistency